libs/storage/Tsavorite/cs/test/BasicTests.cs

```diff
@@ -799,16 +800,17 @@ public static void KVBasicsSampleEndToEndInDocs()
             Assert.AreEqual(10, output);
         }
 
-#if WINDOWS
         [Test]
         [Category("TsavoriteKV")]
         public static void LogPathtooLong()
         {
+            if (!OperatingSystem.IsWindows())
+                Assert.Inconclusive();
```

Comment:
.Inconclusive shows up in VS as "Not Run", so let's make it "Skipped".

Reply:
I think this one should do that then?
```suggestion
                Assert.Ignore("Skipped");
```